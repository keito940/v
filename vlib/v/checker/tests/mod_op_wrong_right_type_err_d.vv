fn main() {
	a := map[string]int
	1 % a
}
