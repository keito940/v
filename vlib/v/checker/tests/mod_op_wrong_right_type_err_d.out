vlib/v/checker/tests/mod_op_wrong_right_type_err_d.v:3:6: error: mismatched types `int` and `map_string_int`
    1| fn main() {
    2|     a := map[string]int
    3|     1 % a
               ^
    4| }
